kernel/packages/shared/session/sagas.ts

```diff
@@ -286,7 +295,7 @@ function* signUp() {
 
   const profile = yield select(getSignUpProfile)
   profile.userId = session.userId.toString()
-  profile.ethAddress = session.userId.toString()
+  profile.ethAddress = session.rawAddress ? session.rawAddress.toString() : profile.userId
```

Comment:
Why do you need to call `toString`?

Reply:
Also, according to the types you defined below this could never be undefined, so this check seems unnecessary

Reply:
I think the same but it was this way before I edited there, I can remove the toString, sure

Reply:
Done